Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -3,6 +3,15 @@ using Uno.Collections;
 
 namespace Fuse.Internal
 {
+	//An error with Uno handling the Enumerator<T> generic is preventing this from being a private enum to MiniList
+	//E2047: No implicit cast from Fuse.Internal.MiniList<T>.Mode to Fuse.Internal.MiniList<T>.Mode
```

Comment:
I think it would be better to file a ticket even if we don't have a minimal reproduction, in case we bother in the future. That way it would be possible to have everything related point to the same place.